connect_bi_reporter/uploads/models.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from sqlalchemy.orm import relationship
+import sqlalchemy as db
+
+from connect_bi_reporter.db import Model
+from connect_bi_reporter.feeds.models import Feed
+from connect_bi_reporter.uploads.enums import UploadStatusChoices
+
+
+class Upload(Model):
+    __tablename__ = 'upload'
+
+    PREFIX = 'ULF'
+    STATUSES = UploadStatusChoices
+
+    id = db.Column(db.String(20), primary_key=True)
+    name = db.Column(db.String(128), nullable=False)
```

Comment:
Let's change it to nullable, because we are going to fill it after upload it's completed. Same for size (set default 0).

Reply:
updated!!